doc/architecture/domains/authorization/features/accept-terms.md

```diff
@@ -0,0 +1,281 @@
+# Terms Acceptance Documentation
+
+## Overview
+
+The Terms Acceptance API allows organizations to accept the latest terms and conditions.
+When an organization accepts the terms, the Authorization database is updated, and an integration event is published,
+to RabbitMQ, using MassTransit's transactional outbox pattern.
+
+## Accept terms flow:
+
+```mermaid
+sequenceDiagram
+    actor User
+    participant API as Terms Endpoint
+    participant Handler as Command Handler
+    participant DB as Database
+    participant RabbitMQ as RabbitMQ Broker
+
+    User->>API: POST /api/authorization/terms/accept
+    activate API
+    API->>API: Validate JWT and extract claims
+    API->>Handler: Send AcceptTermsCommand
+    activate Handler
+
+    rect rgb(50, 50, 50)
+        Note over Handler, DB: Database Interactions
+        Handler->>DB: Begin transaction
+        activate DB
+        Handler->>DB: Get or create organization
+        DB-->>Handler: Return organization
+        Handler->>DB: Fetch latest terms
+        DB-->>Handler: Return latest terms
+        Handler->>DB: Update organization's terms acceptance
+        DB-->>Handler: Confirm update
+        Handler->>DB: Commit transaction
+        DB-->>Handler: Confirm commit
+        deactivate DB
+    end
+
+    rect rgb(50, 50, 50)
+        note over Handler, RabbitMQ: Publish integration event
+        Handler->>RabbitMQ: Send OrgAcceptedTerms message
+    end
```

Comment:
Why the black boxes?

Reply:
I think it looks nice. synes du ikke det gør det nemmere at læse?